Gopkg.lock

```diff
@@ -57,7 +57,7 @@
     "pkg/clientset/versioned/typed/core/v1alpha1",
     "pkg/clientset/versioned/typed/provider/v1alpha1"
   ]
-  revision = "75c93359f047754a11e598787fb09d20b99745ca"
+  revision = "f0b8bcd6122deaf8be38b9705ac528cf4b749e3a"
```

Comment:
Vendor update did not help. The upstream issue still persists. When digging around they plan to provide this functionality but this can take quite some time.